nixos/modules/installer/tools/tools.nix

```diff
@@ -10,119 +10,163 @@ let
     dir = "bin";
     isExecutable = true;
   });
```

Comment:
Could you please split the refactoring into a separate commit? It is currently difficult to review the diff.

Reply:
@jtojnar done. I'm afraid the diff of the second commit is still quite large though, since things moved around quite a bit.